.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 4
+
+[{package.json,.travis.yml}]
+indent_style = space
+indent_size = 2
+
+[Makefile]
+indent_style = tab
```

Comment:
@ju1ius Can you please remove it? It will be auto updated by dev-kit later: https://github.com/sonata-project/dev-kit/blob/master/tools/editorconfig/.editorconfig

Feel free to contribute on the following link if necessary. :+1: